server/lib/initial-wazuh-config.ts

```diff
@@ -27,211 +63,226 @@ export const initialWazuhConfig: string = `---
-#hideManagerAlerts: false
+# ---------------------------- Hide manager alerts -----------------------------
 #
-# ------------------------------- App logging level -----------------------------
-# Set the logging level for the Wazuh App log files.
-# Default value: info
-# Allowed values: info, debug
-#logs.level: info
+${splitDescription(configEquivalences.hideManagerAlerts)}
+# hideManagerAlerts: ${WAZUH_DEFAULT_APP_CONFIG.hideManagerAlerts}
 #
-# -------------------------------- Enrollment DNS -------------------------------
-# Set the variable WAZUH_REGISTRATION_SERVER in agents deployment.
-# Default value: ''
-#enrollment.dns: ''
+# ------------------------------- App logging level ----------------------------
 #
-# Wazuh registration password
-# Default value: ''
-#enrollment.password: ''
-#-------------------------------- API entries -----------------------------------
-#The following configuration is the default structure to define an API entry.
+${splitDescription(configEquivalences['logs.level'])}
+# Allowed values are: info, debug
+# logs.level: ${WAZUH_DEFAULT_APP_CONFIG['logs.level']}
 #
-#hosts:
-#  - <id>:
-      # URL
-      # API url
-      # url: http(s)://<url>
-
-      # Port
-      # API port
-      # port: <port>
-
-      # Username
-      # API user's username
-      # username: <username>
-
-      # Password
-      # API user's password
-      # password: <password>
+# ------------------------------- Agent enrollment -----------------------------
+#
+${splitDescription(configEquivalences['enrollment.dns'])}
+# enrollment.dns: ${WAZUH_DEFAULT_APP_CONFIG['enrollment.dns']}
+#
+${splitDescription(configEquivalences['enrollment.password'])}
+# enrollment.password: ${WAZUH_DEFAULT_APP_CONFIG['enrollment.password']}
+#
+#-------------------------------- Wazuh hosts ----------------------------------
+#
+# The following configuration is the default structure to define a host.
+#
+# hosts:
+# # Host ID / name,
+#   - env-1:
+#     # Host URL
+#     url: https://env-1.example
+#     # Host / API port
+#     port: 55000
+#     # Host / API username
+#     username: wazuh-wui
+#     # Host / API password
+#     password: wazuh-wui
+#     # Use RBAC or not. If set to true, the username must be "wazuh-wui".
+#     run_as: true
+#   - env-2:
+#     url: https://env-2.example
+#     port: 55000
+#     username: wazuh-wui
+#     password: wazuh-wui
+#     run_as: true
 
-      # Run as
-      # Define how the app user gets his/her app permissions.
-      # Values:
-      #   - true: use his/her authentication context. Require Wazuh API user allows run_as.
-      #   - false or not defined: get same permissions of Wazuh API user.
-      # run_as: <true|false>
 hosts:
   - default:
-     url: https://localhost
-     port: 55000
-     username: wazuh-wui
-     password: wazuh-wui
-     run_as: false
-`
+    url: https://localhost
+    port: 55000
+    username: wazuh-wui
+    password: wazuh-wui
+    run_as: false
```

Comment:
**issue**: I think there is a problem with the identation of these settings that causes an error and when accessing the Wazuh plugin I get:
![image](https://user-images.githubusercontent.com/34042064/182843655-3ba07ec9-4ea5-409f-abd9-3c10de1dd267.png)

**suggestion**: Add the spaces that were removed.

If we check this is the problem, then it should modify the examples above too.

Reply:
The indentation problem has been fixed in https://github.com/wazuh/wazuh-kibana-app/pull/4378/commits/00b9f037dadbb727065f7da07fd6777941244b23.

Please re-review the changes.

Thank you!